### ISSUE TITLE
Failed create configDir

### ISSUE THREAD

**Issue by Arslanbekov** [aerokube/cm#34]
Before run `cm` in docker i have error: 
```
Failed to start Selenoid: failed to create container: Error response from daemon: create .aerokube/selenoid: ".aerokube/selenoid" includes invalid characters for a local volume name, only "[a-zA-Z0-9][a-zA-Z0-9_.-]" are allowed. If you intented to pass a host directory, use absolute path
```

Command: 
```
docker run --rm -e DOCKER_API_VERSION=1.24 -v /var/run/docker.sock:/var/run/docker.sock aerokube/cm:latest selenoid start
```

**Comment by vania-pooh** [comment]
What is the exact Docker version are you using?

**Comment by Arslanbekov** [comment]
@vania-pooh 
on docker version:

```
Client:
 Version:      1.12.6
 API version:  1.24
 Go version:   go1.6.4
 Git commit:   78d1802
 Built:        Tue Jan 10 20:38:45 2017
 OS/Arch:      linux/amd64

Server:
 Version:      1.12.6
 API version:  1.24
 Go version:   go1.6.4
 Git commit:   78d1802
 Built:        Tue Jan 10 20:38:45 2017
 OS/Arch:      linux/amd64
```

**Comment by vania-pooh** [comment]
Ok, will try to reproduce.

**Comment by vania-pooh** [comment]
Yes, this is reproducing. Will fix soon.

**Comment by vania-pooh** [comment]
Fixed this in #37. Correct command is:
```
# docker run --rm -v /var/run/docker.sock:/var/run/docker.sock -v /root:${HOME} aerokube/cm:latest selenoid start
```
Please try with latest container when merged.
